packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -70,5 +70,11 @@ export const messages = {
   force_lightning_app_create_text: 'SFDX: Create Lightning App',
   force_lightning_component_create_text: 'SFDX: Create Lightning Component',
   force_lightning_event_create_text: 'SFDX: Create Lightning Event',
-  force_lightning_interface_create_text: 'SFDX: Create Lightning Interface'
+  force_lightning_interface_create_text: 'SFDX: Create Lightning Interface',
+  warning_prompt_file_overwrite:
+    'File(s) with specified path already exists in workspace. Would you like to overwrite?',
+  warning_prompt_lightning_bundle_overwrite:
```

Comment:
@ruthemmanuelle Could you check this new message? Context is that Lightning is a little different than the other create commands since it will place the files into a new directory with the name specified by the user.

We want to warn the user of 2 things with the message:
1) This directory is already in use by a lightning bundle (event, cmp, app, interface)
2) Depending on the bundle being created, some files may be overwritten since there are overlap in the files being created. See attached screenshot for an example.

![image](https://user-images.githubusercontent.com/4503716/30300396-4a2af980-9709-11e7-8739-0fc228d64c9d.png)

/cc @vazexqi